core/store/orm/schema.go

```diff
@@ -42,6 +42,7 @@ type ConfigSchema struct {
 	GasUpdaterEnabled               bool            `env:"GAS_UPDATER_ENABLED" default:"false"`
 	JSONConsole                     bool            `env:"JSON_CONSOLE" default:"false"`
 	LinkContractAddress             string          `env:"LINK_CONTRACT_ADDRESS" default:"0x514910771AF9Ca656af840dff83E8264EcF986CA"`
+	LogBackfillDepth                string          `env:"LOG_BACKFILL_DEPTH" default:"10"`
```

Comment:
Maybe rename it BLOCK_BACKFILL_DEPTH, or give it something to indicate what the unit is that the setting is measured in?

Reply:
:+1: yeah that sounds better to me